.github/workflows/R-CMD-check.yaml

```diff
@@ -87,6 +87,11 @@ jobs:
           vaultr::vault_test_server_install()
         shell: Rscript {0}
 
+      - name: Install devl RSQlite
```

Comment:
This can be removed now, new package is on cran :tada:

https://cran.r-project.org/web/packages/RSQLite/index.html - just waiting on windows r-release but that should be ok